cmd/image-builder-maintenance/db.go

```diff
@@ -114,23 +113,32 @@ func (d *maintenanceDB) LogVacuumStats(ctx context.Context) (int64, error) {
 				return int64(0), err
 			}
 
-			logrus.WithFields(logrus.Fields{
-				"table_name":        relName,
-				"table_size":        relSize,
-				"tuples_inserted":   ins,
-				"tuples_updated":    upd,
-				"tuples_deleted":    del,
-				"tuples_live":       live,
-				"tuples_dead":       dead,
-				"vacuum_count":      vc,
-				"autovacuum_count":  avc,
-				"last_vacuum":       lvc,
-				"last_autovacuum":   lavc,
-				"analyze_count":     ac,
-				"autoanalyze_count": aac,
-				"last_analyze":      lan,
-				"last_autoanalyze":  laan,
-			}).Info("Vacuum and analyze stats for table")
+			attrs := []any{
```

Comment:
Just **out of interest**: Do we need to build an array of attributes or is it possible to inline it to With and this is just personal choice?

Reply:
It cannot be done, it is limitation of the Go compiler one cannot mix variadic arguments with normal arguments. I In other words, this is not possible, at least to my knowledge:

```
fmt.Println("This", "is", "okay")
	
slice := []string{"is", "not"}
fmt.Println("But", "this", slice...) // compiler error
```

Playground: https://go.dev/play/p/ewHtdzD6pm8